src/main/resources/filter/web/positioning.filter

```diff
@@ -0,0 +1,94 @@
+# Positioning filter file for recheck. Please do not delete, even if it is empty.
+
+# You can ignore specific attributes like so: 
+# matcher: retestid=div-b4f23, attribute: font.*
+
+# To ignore whole subtrees, use e.g. xpath:
+# matcher: xpath=HTML[1]/BODY[1]/DIV[1]/DIV[1]/DIV[1]
+
+# To ignore attributes globally, use 
```

Comment:
No punctuation (`:` or `.`) at the end.

Reply:
I will fix that, thanks. But i copied the comments from the recheck.ignore file and the punctiation, space, case  there should be changed too. Should i do that?

Reply:
Yes, please fix that in the "Master"-recheck.ignore file. Since these files are "internal", I don't think the comments are sensible here anyway – please remove them here.